src/link.tsx

```diff
@@ -18,11 +18,11 @@ export type RegisteredLinkProps = RegisterLink extends { Link: infer Link }
           | Omit<UnpackProps<Link>, "children">
           | (Omit<HTMLAnchorProps, "children" | "href"> & {
                 href:
-                    | `mailto:${string}`
                     | `//${string}`
                     | `https://${string}`
                     | `http://${string}`
-                    | `#${string}`;
+                    | `#${string}`
+                    | string; // to handle mailto:mail@domain.fr or tel:0123456789  ...
```

Comment:
Forget about the comment, I'll write it later, just use `string` all other types are swalowed anyway.
```ts
href: string;
```